source/gui/nvdaControls.py

```diff
@@ -376,6 +377,56 @@ def _onShowEvt(self, evt):
 		evt.Skip()
 
 
+class ContinueCancelDialog(MessageDialog):
```

Comment:
Considering we want to amalgamate this into a single class via #13007 in this release, I think this should be a private class. I don't think we intend to release this dialog in 2025.1. @SaschaCowley - thoughts?

Reply:
I was unaware that was targeted for action soon. This can easily be made private.
I could even make it self-contained (from `MessageDialog`, at least), and change the implementation in some future PR.

Reply:
I appreciate it has stalled a lot - there's also a chance it gets stalled out of this release, but it is planned for 2025.1. I'll hold off on Sascha's thoughts on how to proceed here as she's picked up the work from me (very thankful for that)

Reply:
I support making this a private class. I think it worthwhile to keep it here so this work can be merged, just in case the message dialog API work doesn't end up making it to 2025.1. Also increases the test case surface for the new API, which is helpful.